src/react-view/contents/graphs/graph-with-rechart/view.tsx

```
@@ -16,31 +16,32 @@ import {
 const genericToRecharts = <K extends string, SK extends string>(
     inputData: DSDoubleGroup<K, SK>[],
-): { data: Array<Record<string, number>>; firstKey: string; keys: SK[] } => {
+): { data: Record<string, ApiUUID>[]; firstKey: string; keys: SK[] } => {
     const keys: Set<SK> = new Set();
     const [first] = inputData;
     const data = inputData.map(group => ({
         [group.groupName]: group.label,
-        ...group.value.reduce((acc, cur) => {
+        ...group.value.reduce<Partial<Record<SK, ApiUUID>>>((acc, cur) => {
             keys.add(cur.label);
             return { ...acc, [cur.label]: cur.value };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)